### DISCUSSION TITLE
Pimoroni NVMe for Pi5

### DISCUSSION THREAD

**Discussion by Marmachine** [geerlingguy/raspberry-pi-pcie-devices#574] [General] [answered]
Have you seen this one?
https://shop.pimoroni.com/products/nvme-base?variant=41219587211347

Ordered one, expecting end Jan to be operational

**Comment by geerlingguy** [comment, marked as answer]
Yep! I have one and will be testing it soon! https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/565

**Comment by Marmachine** [comment]
Sorry, must have missed #565
Curious about your results then.
Since it's mounted to the back of the Pi it leaves space for ventilation and (hopefully soon) a POE-hat!
